resources/cmake/FindRTIConnextDDS.cmake

```diff
@@ -27,10 +27,10 @@
 # find_package invocation for this module to set variables for the different
 # libraries associated with them.
 #
-# Also, the version consistency across all the requested components will be
+# Also, the version consistency across all the requested components can be
 # checked. If the version between the components missmatch,
-# `RTIConnextDDS_FOUND` will be set to `FALSE`. This feature can be disabled
-# setting `CONNEXTDDS_DISABLE_VERSION_CHECK` to `TRUE`.
+# `RTIConnextDDS_FOUND` will be set to `FALSE`. This feature can be enabled
+# setting `CONNEXTDDS_ENABLE_VERSION_CHECK` to `TRUE`.
 #
```

Comment:
Maybe we want to call this ENABLE_VERSION_CONSISTENCY_CHECK. If I understand this correctly, we always check we're using the right version using a header file, but this check is actually going a bit further and parsing rti_versions.xml, right? Maybe we should be a bit more explict about it.